### DISCUSSION TITLE
Github: The discussions tab doesn't have a button to start a discussion

### DISCUSSION THREAD

**Discussion by SenH** [dsmrreader/dsmr-reader#1786] [Questions & answers / How-to's] [answered]
### Description

Is the discussions tab still active (or disabled for non contributors)?
![image](https://user-images.githubusercontent.com/168986/210765110-1b713ff3-d2cf-470d-a9ae-29058d3af9a9.png)

### DSMR-reader version

v5.9

### DSMR-reader platform

Docker (e.g. Xirixiz)

**Comment by dennissiemensma** [comment, marked as answer]
Only announcements _should_ be disabled for public, the other one I'd expect to work. But not sure...

----

![Screenshot from 2023-01-05 20-45-21](https://user-images.githubusercontent.com/16581744/210866970-af519de4-2dc1-4425-af2c-481b63f101a4.png)

---

> **Question / Answer**
> Enable your community to ask questions, suggest answers, and vote on the best suggested answer.

**Reply by dennissiemensma** [reply]
And copying/moving seems to work, but I cannot verify if it works as regular github user though

**Reply by dennissiemensma** [reply]
I will try the other category, which does not require "answers".

**Reply by dennissiemensma** [reply]
I've created a new "Open-ended discussions" category, you can try that @SenH

**Reply by SenH** [reply]
the green "start discussion" button is still missing? I can see it on other repos. Not sure what is happening :)
<img width="1286" alt="Screenshot 2023-01-08 at 21 02 07" src="https://user-images.githubusercontent.com/168986/211216441-7f2627f4-440d-4069-8b59-27858980dcdf.png">

It should look like this:
<img width="1290" alt="Screenshot 2023-01-08 at 21 03 29" src="https://user-images.githubusercontent.com/168986/211216547-a84c6889-031d-468e-8da1-e606c284b677.png">

**Reply by dennissiemensma** [reply]
I'm not sure what else to check, it should be enabled.

![Screenshot from 2023-01-08 21-32-53](https://user-images.githubusercontent.com/16581744/211217716-ea3dcd4a-5d8f-4b3c-a36a-da0da27dce90.png)

**Reply by SenH** [reply]
Yeah, I'm clueless also.

**Comment by sts-ux** [comment]
Hello,

Is the adding discussions disabled? I see Topics in Discussions tab but nowhere to add a new one. Therefore I'm asking questions via Issues which actually might not be needed there.

Thanks!

**Reply by dennissiemensma** [reply]
I have no idea, it should be enabled: 
<img width="1501" height="1302" alt="image" src="https://github.com/user-attachments/assets/9f193c08-2a9d-4cd2-b45f-e44647f5e145" />

**Comment by sts-ux** [comment]
I see,

Tried Firefox and Edge and do not see option to add item to discuss.
Any of the forum users knows the answer?
Thank you!

<img width="1249" height="753" alt="image" src="https://github.com/user-attachments/assets/c96668eb-858d-4d74-8f16-ae6c95f782ef" />

**Reply by dennissiemensma** [reply]
Ah I found it, it was enabled on the repo, but not the org.

**Reply by dennissiemensma** [reply]
<img width="2040" height="570" alt="Screenshot from 2026-02-15 20-12-56" src="https://github.com/user-attachments/assets/ca22307c-8858-421e-991c-e21ecb12a8a0" />


----


<img width="2040" height="1076" alt="Screenshot from 2026-02-15 20-13-06" src="https://github.com/user-attachments/assets/f90a533e-bbb3-40cd-9398-a5643fda6cf2" />

**Reply by dennissiemensma** [reply]
This repo used to be on my personal account, never knew this was a difference

**Comment by sts-ux** [comment]
Still not possible to add items to discuss :(

**Reply by dennissiemensma** [reply]
Maybe now? This was off somewhere in another screen hidden away:

<img width="1665" height="334" alt="image" src="https://github.com/user-attachments/assets/0474b9df-c017-4804-b184-63a2bca235ba" />

**Comment by sts-ux** [comment]
Works now!
